Documentation/network/bgp-control-plane-operation.rst

```diff
@@ -68,6 +68,36 @@ forwarded to a different node than before, will be reset.
 
 .. _Resilient Hashing: https://www.juniper.net/documentation/us/en/software/junos/interfaces-ethernet-switches/topics/topic-map/switches-interface-resilient-hashing.html
 
+Node Down
+---------
+
+If the node goes down, the BGP session will be lost, and the Cilium datapath
+forwarding will be stopped. If the node is shut down gracefully with a proper
+BGP shutdown procedure or TCP connection close, the BGP peer either stops
+forwarding immediately or takes some time to stop forwarding traffic to the
+node depending on the Graceful Restart settings. Since the Cilium datapath is
+stopping, the traffic will be dropped during this period. This is especially
+problematic when you advertise the route to the Service with
```

Comment:
I think an explanation of why this is problematic is useful. 

BGP peer will continue to forward traffic to this node even if BGP session goes down due to node shutdown till GR restart timer expires.

Reply:
Made a `Involuntary Shutdown` section. I also added the `Voluntary Shutdown` section and referred to the `Node Shutdown` section from it.

Reply:
Yep, documented it.